### ISSUE TITLE
Filter contacts on ID or based on "source" field for CSV imports

### ISSUE THREAD

**Issue by lightweight** [mautic/mautic#2274]
Feature request

## Description:
I have just imported a large set (2000+) contacts from a CSV. I would love to be able to do the following: 
1. have a field automatically added to this contacts called "source" with the CSV filename from which they were imported (because, logically, I want them all to be in the same segment, but I haven't been able to find another way to identify them effectively, as they only have First, Last name, and email address fields). 
2. I know they were added as Contact id 99-2100... I'd love to be able to filter based on id, e.g. a grammar like ```id:99-2100``` or ```id is greater than or equal to 99```

Perhaps those types of filters are possible already, but I wasn't able to work out how.

**Comment by alanhartless** [comment]
Hi!

There are two ways to do this currently. On the import page - there's an option at the top to select the segments to automatically add those imported to. No need for a filter.

![](http://alan.direct/drop/2016-08-16_18-01-57.png)

The second option is to use a tag when importing that you could then base a filter on. (To the right of the segments dropdown in that screenshot).

Would either of those meet your needs? You can safely reimport choosing either of those as long as you have the email field matched up again. 

Thanks!

**Comment by lightweight** [comment]
Very useful, thanks for the insight (that I can re-import as long as emails match)! I'll try the re-import and segment assignment (I'm hoping I can assign to more than one segment. When I first imported the list, I thought I was setting a tag at the time of import (i.e. I set the interface to do that), but following the import the tag wasn't associated with any of the contacts... Anyway, that was my first contact import attempt, so will try again.
